Adjust the Lid Parameter on the Rate Limited Flapper, May 16, 2022.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Adjust the Lid Parameter on the Rate Limited Flapper, May 16, 2022
+summary: Signal your support or opposition to Adjust the Lid Parameter on the Rate Limited Flapper
+discussion_link: https://forum.makerdao.com/t/signal-request-change-lid-parameter-on-the-rate-limiter/14688
+vote_type: Plurality Voting
+categories:
+   - Risk Variable
+   - Technical
+   - Auctions
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-05-1616:00:00
+end_date: 2022-05-1916:00:00
+---
+# Poll:Adjust the Lid Parameter on the Rate Limited Flapper, May 16, 2022
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the MakerDAO Community. This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, May 16 at 16:00 UTC.
+
+**This is a binary vote. In a binary vote, you may vote for a single option. You should vote for the option which you prefer. Alternatively, if you would accept either of the two options and have no preference between them, you should vote 'Abstain'.**
```

Comment:
Can you use the vote description from the template? link - https://github.com/makerdao/community/blob/master/governance/polls/templates/Generic/$GenericPlurality%20-%20$date_MONTH_DD,_YYYY.md?plain=1

Should be:

**This is a plurality vote.** 
- **You may vote for a single option.**
- **You should vote for the option that you prefer.** 
- **If you would accept any of the options you should vote 'Abstain'.**